tests/test_arithmetic.py

```diff
@@ -0,0 +1,90 @@
+"""
+"""
+import pytest
+
+import calc
+
+
+def test_parse_subtraction():
```

Comment:
It feels like repetition to me. I have a weak opinion against as long as the names make sense.